app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,23 @@
+class NotificationsController < ApplicationController
+  before_action :set_room
+
+  def index
+    respond_to do |format|
+      format.json { render json: @notifications }
```

Comment:
is it going to be only `json` for the controller? if that the case, we can default the format to `json` (in either routes or controller) and don't use respond to